src/data/vapor-pressure-constants.ts

```diff
@@ -3629,6 +3629,1551 @@ export const DIPPRVaporPressureDictionary: Record<string, DIPPRVaporPressureProp
+    C5: 6,
+    minimumTemperature: 234.18,
+    maximumTemperature: 560.95,
+    pressureAtMinimumTemperature: 7.34e1,
+    pressureAtMaximumTemperature: 3.699e6
+  },
+
+  "1Pentene": {
+    name: "1-Pentene",
+    formula: "C5H10",
+    CAS: "109-67-1",
+    C1: 46.994,
+    C2: -4289.5,
+    C3: -3.7345,
+    C4: 2.54e-17,
+    C5: 6,
+    minimumTemperature: 108.02,
+    maximumTemperature: 464.8,
+    pressureAtMinimumTemperature: 3.71e-5,
+    pressureAtMaximumTemperature: 3.562e6
+  },
+
+  "2PentylMercaptan": {
+    name: "2-Pentyl mercaptan",
+    formula: "C5H12S",
+    CAS: "2084-19-7",
+    C1: 58.985,
+    C2: -6193.1,
+    C3: -5.2746,
+    C4: 7.4e-18,
+    C5: 6,
+    minimumTemperature: 160.75,
+    maximumTemperature: 584.3,
+    pressureAtMinimumTemperature: 1.77e-3,
+    pressureAtMaximumTemperature: 3.537e6
+  },
+
+  PentylMercaptan: {
+    name: "Pentyl mercaptan",
+    formula: "C5H12S",
+    CAS: "110-66-7",
+    C1: 67.309,
+    C2: -6880.8,
+    C3: -6.4449,
+    C4: 1.01e-17,
+    C5: 6,
+    minimumTemperature: 197.45,
+    maximumTemperature: 598,
+    pressureAtMinimumTemperature: 0.201,
+    pressureAtMaximumTemperature: 3473000
+  },
+
+  "1Pentyne": {
+    name: "1-Pentyne",
+    formula: "C5H8",
+    CAS: "627-19-0",
+    C1: 82.805,
+    C2: -5683.8,
+    C3: -9.4301,
+    C4: 0.000010767,
+    C5: 2,
+    minimumTemperature: 167.45,
+    maximumTemperature: 481.2,
+    pressureAtMinimumTemperature: 2.4,
+    pressureAtMaximumTemperature: 4.17e6
+  },
+
+  "2Pentyne": {
+    name: "2-Pentyne",
+    formula: "C5H8",
+    CAS: "627-21-4",
+    C1: 137.29,
+    C2: -7447.1,
+    C3: -19.01,
+    C4: 0.021415,
+    C5: 1,
+    minimumTemperature: 163.83,
+    maximumTemperature: 519,
+    pressureAtMinimumTemperature: 2.05e-1,
+    pressureAtMaximumTemperature: 4.02e6
+  },
+
+  Phenanthrene: {
+    name: "Phenanthrene",
+    formula: "C14H10",
+    CAS: "85-01-8",
+    C1: 72.958,
+    C2: -10943,
+    C3: -6.7902,
+    C4: 1.09e-18,
+    C5: 6,
+    minimumTemperature: 372.38,
+    maximumTemperature: 869,
+    pressureAtMinimumTemperature: 2.93e1,
+    pressureAtMaximumTemperature: 2.902e6
+  },
+
+  Phenol: {
+    name: "Phenol",
+    formula: "C6H6O",
+    CAS: "108-95-2",
+    C1: 95.444,
+    C2: -10113,
+    C3: -10.09,
+    C4: 6.76e-18,
+    C5: 6,
+    minimumTemperature: 314.06,
+    maximumTemperature: 694.25,
+    pressureAtMinimumTemperature: 1.88e2,
+    pressureAtMaximumTemperature: 6.058e6
+  },
+
+  PhenylIsocyanate: {
+    name: "Phenyl isocyanate",
+    formula: "C7H5NO",
+    CAS: "103-71-9",
+    C1: 86.779,
+    C2: -8101.8,
+    C3: -9.5303,
+    C4: 6.1367e-6,
+    C5: 2,
+    minimumTemperature: 243.15,
+    pressureAtMinimumTemperature: 4.33,
+    maximumTemperature: 653,
+    pressureAtMaximumTemperature: 4.063e6
+  },
+
+  PhthalicAnhydride: {
+    name: "Phthalic anhydride",
+    formula: "C8H4O3",
+    CAS: "85-44-9",
+    C1: 126.5,
+    C2: 12551,
+    C3: 15.002,
```

Comment:
−15.002

Reply:
Before line 4579 changes are made in batch10